python/lsst/daf/persistence/deprecation.py

```diff
@@ -42,7 +42,8 @@
 # This is the warning message to issue. There is a "label" placeholder
 # that should be inserted on format.
 _warning_msg = "Gen2 Butler has been deprecated{label}. It will be removed sometime after v23.0 but no" \
-               " earlier than the end of 2021."
+               " earlier than the end of 2021." \
```

Comment:
I'd say:
```
Gen2 Butler has been deprecated{label}.  This Gen2 code may be removed in any future daily or weekly release.
```
It won't stop working in a given current release.  We shouldn't have it be present but broken.  And I don't think the dates say much at this point.